manifests/wftmpl.yaml

```diff
@@ -1,63 +1,66 @@
 apiVersion: argoproj.io/v1alpha1
 kind: WorkflowTemplate
 metadata:
-  name: <CHANGE_ME>
+  name: ocp-ci-analysis
 spec:
   entrypoint: entrypoint
   templates:
-    # - name: entrypoint
-    #   steps:
-    #     - - name: data-collection
-    #         template: data-collection
-    #     - - name: analysis
-    #         template: analysis
-    #
-    # - name: data-collection
-    #   dag:
-    #     tasks:
-    #       - name: collect-raw-data
-    #         template: notebook-executor
-    #         arguments:
-    #           parameters: [{name: notebook, value: <PATH_WITHIN_NOTEBOOKS_FOLDER>}]
-    #       - ...
-    #
-    # - name: analysis
-    #   steps:
-    #     - - name: contributor-analysis
-    #         template: notebook-executor
-    #         arguments:
-    #           parameters: [{name: notebook, value: <PATH_WITHIN_NOTEBOOKS_FOLDER>}]
-    #       - ...
+    - name: entrypoint
+      steps:
+        - - name: metrics
+            template: metrics
+
+    - name: metrics
+      dag:
+        tasks:
+          - name: number-of-flakes
+            template: notebook-executor
+            arguments:
+              parameters:
+                [{ name: notebook, value: data-sources/TestGrid/number_of_flakes.ipynb }]
+
+          - name: persistent-failures
+            template: notebook-executor
+            arguments:
+              parameters:
+                [{ name: notebook, value: data-sources/TestGrid/persistent_failures_analysis.ipynb}]
+
+          - name: test-pass-failures
+            template: notebook-executor
+            arguments:
+              parameters:
+                [{ name: notebook, value: data-sources/TestGrid/test_pass_failures.ipynb}]
```

Comment:
You don't need a dag + steps if you're just executing things in parallel... You can do it via steps directly and in a readable way:

https://argoproj.github.io/argo-workflows/workflow-concepts/#steps

Reply:
I expect to have dependencies between steps in the near future. My understanding is that a dag w/o dependencies will work as needed for now.

Reply:
yes. The dag without dependencies will execute all the tasks in parallel - if that's the intended behavior, than it's fine.

I would choose an approach of keeping it tidy (having just 3 parallel steps like this for now and expand on it in future rather than preparing for something that might or might not happen.

```yaml
      steps:
        - - name: number-of-flakes
            template: notebook-executor
            arguments:
              parameters:
                [{ name: notebook, value: data-sources/TestGrid/number_of_flakes.ipynb }]

          - name: persistent-failures
            template: notebook-executor
            arguments:
              parameters:
                [{ name: notebook, value: data-sources/TestGrid/persistent_failures_analysis.ipynb}]

          - name: test-pass-failures
            template: notebook-executor
            arguments:
              parameters:
                [{ name: notebook, value: data-sources/TestGrid/test_pass_failures.ipynb}]
```

But that's your call. It's now wrong either way. :slightly_smiling_face:

Reply:
Ok, changed as you wrote.